Stellar-ledger.x

```diff
@@ -400,10 +400,27 @@ struct DiagnosticEvent
     ContractEvent event;
 };
 
-struct SorobanTransactionMeta 
+struct SorobanTransactionMetaExtV1
 {
     ExtensionPoint ext;
 
+    int64 totalNonRefundableResourceFeeSpent;
+    int64 totalRefundableResourceFeeSpent;
```

Comment:
Isn't `totalRefundableResourceFeeSpent` == `rentFeeSpent` + fees spent on the events/return value? Should this just be the events/return fee? Or is the idea here that it could contain other components we may add in the future?

Reply:
Yeah, I think it's a bit more future-proof - one can always rely on the total refundable fee and `feeCharged - nonRefundableFee - refundableFee` equation. If we add more refundable components in the future we may or may not expose them in meta, but the existing logic won't need to be changed.

Reply:
Makes sense. I think it'd be helpful if we added a comment about the relationship between `totalRefundableResourceFeeSpent`, `rentFeeSpent `, and the events.

Reply:
I've added documentation regarding the most necessary equations to keep in mind when using this.